homeassistant/components/enigma2/media_player.py

```diff
@@ -81,49 +81,44 @@ async def async_setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up of an enigma2 media player."""
-    if discovery_info:
-        # Discovery gives us the streaming service port (8001)
-        # which is not useful as OpenWebif never runs on that port.
-        # So use the default port instead.
-        config[CONF_PORT] = DEFAULT_PORT
-        config[CONF_NAME] = discovery_info["hostname"]
-        config[CONF_HOST] = discovery_info["host"]
-        config[CONF_USERNAME] = DEFAULT_USERNAME
-        config[CONF_PASSWORD] = DEFAULT_PASSWORD
-        config[CONF_SSL] = DEFAULT_SSL
-        config[CONF_USE_CHANNEL_ICON] = DEFAULT_USE_CHANNEL_ICON
-        config[CONF_MAC_ADDRESS] = DEFAULT_MAC_ADDRESS
-        config[CONF_DEEP_STANDBY] = DEFAULT_DEEP_STANDBY
-        config[CONF_SOURCE_BOUQUET] = DEFAULT_SOURCE_BOUQUET
-
-    base_url = URL.build(
-        scheme="https" if config[CONF_SSL] else "http",
-        host=config[CONF_HOST],
-        port=config.get(CONF_PORT),
-        user=config.get(CONF_USERNAME),
-        password=config.get(CONF_PASSWORD),
-    )
 
-    session = async_create_clientsession(hass, verify_ssl=False, base_url=base_url)
+    entry_data = {
```

Comment:
The mac address configured for wake on LAN is not imported, should that be mentioned in the breaking change section? Although based on the code it seems like that functionality was not working?

Reply:
This functionality is not working for quite some time now and the MAC address of the device is recieved from the device itself via the API.

The question is, if it is a good idea to implement it again? Because Home Assistant has a `wake_on_lan` integration, and nobody ever created an issue for this integration about missing the functionality.

Reply:
That's up to you. It's probably OK to rely on `wake_on_lan` instead of duplicating that functionality.